AnimeWordle/demo/src/main/java/com/example/demo/model/Character.java

```diff
@@ -0,0 +1,88 @@
+package com.example.demo.model;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+@Entity
+public class Character{
+    @Id
+    private Long id;
+    private String name;
+    private String nameOfSeries;
+    private boolean gender;
+    private String hairColor;
+    private String specialAbilities;
+    private String affiliation;
+
+    public Character(){
+
```

Comment:
Maybe we can set default values for the character class.

Call other constructor and fill in values